helm/values-prod.yaml

```diff
@@ -133,8 +136,8 @@ microservice-chart:
     CLOSE_PAYMENT_RETRY_EVENT_MAX_ATTEMPTS: "3"
     NOTIFICATION_RETRY_EVENT_BASE_INTERVAL_SECONDS: "600"
     NOTIFICATION_RETRY_EVENT_MAX_ATTEMPTS: "3"
-    AUTHORIZATION_REQUESTED_RETRY_EVENT_BASE_INTERVAL_SECONDS: "60"
-    AUTHORIZATION_REQUESTED_RETRY_EVENT_MAX_ATTEMPTS: "3"
+    AUTHORIZATION_OUTCOME_WAITING_EVENT_BASE_INTERVAL_SECONDS: "60"
```

Comment:
maybe this base interval value have to be lowered? what do you think?

Reply:
Remain only this parameter value to be discussed as 1 minute retry offset maybe is too large given the actual f.e. polling timeout

Reply:
Se to 10. Passible of future evaluation.